src/extension.js

```diff
@@ -180,19 +180,59 @@ function OptionSection(storage, defaultValues) {
 }
 
 /**
- * @param {NodeList|Node[]} nodes The collection of the anchor nodes representing search result to move focus for.
+ * @param {...[Element[], function|null]} var_args The array of tuples.
+ * Each tuple contains collection of the search results optionally accompanied
+ * with their container selector.
  * @constructor
  */
-function SearchResults(nodes) {
-  this.items = Array.prototype.slice.call(nodes);
+function SearchResultCollection(var_args) {
```

Comment:
use varArgs?

Reply:
I based in on https://stackoverflow.com/questions/4729516/correct-way-to-document-open-ended-argument-functions-in-jsdoc. If you think we should not use this notation, I will remove the var_args parameter.

Reply:
Got it. I would prefer to either remove it or stick to the Google style guide (see section 5.5.5.2): https://google.github.io/styleguide/jsguide.html#features-functions-parameters

Reply:
So I should use `...var_args`? I was afraid of compatibility, as it is ES6 feature, isn't it?

Reply:
Yes I believe it's ES6 only, but the compatibility seems pretty good: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters

Specifically Chrome, Firefox, and Edge all support it for at least 2 years, so I'm ok with using that feature.